redfish-core/lib/roles.hpp

```diff
@@ -119,14 +119,23 @@ inline void requestRoutesRoles(App& app)
                 asyncResp->res.jsonValue = {
                     {"@odata.type", "#Role.v1_3_0.Role"},
                     {"Name", "User Role"},
-                    {"Description", roleId + " User Role"},
+                    {"Description", ""},
```

Comment:
This line is not needed if Description is set below.  Alternately, set Description to `roleId`.

Reply:
Yeah, let's get rid of this line. I don't like setting it 2 places in the JSON